api/platform/v1/types.go

```diff
@@ -361,6 +361,8 @@ type ClusterFeature struct {
 	// +optional
 	EnableMetricsServer bool `json:"enableMetricsServer,omitempty" protobuf:"bytes,12,opt,name=enableMetricsServer"`
 	// +optional
+	EnableCilium bool `json:"enableCilium,omitempty" protobuf:"bytes,12,opt,name=enableCilium"`
+	// +optional
```

Comment:
protobuf 12 already get used,  you code will cause serialization and deserialization failed